cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedFunction.qll

```diff
@@ -214,7 +214,7 @@ class TranslatedFunction extends TranslatedRootElement, TTranslatedFunction {
         exists(ThrowExpr throw | throw.getEnclosingFunction() = func)
         or
         exists(FunctionCall call | call.getEnclosingFunction() = func |
-          getTranslatedExpr(call).(TranslatedCallExpr).mayThrowException()
+          getTranslatedExpr(call).(TranslatedCallExpr).mayRaiseException(_)
```

Comment:
`mayRaiseException` doesn't imply, `alwaysRaiseException`, right? If not, we should probably also emit an `Unwind` instruction if `alwaysRaiseException` holds

Reply:
may raise is exclusive of always raise.

Reply:
I misunderstood the original concern. I'll fix this.

Reply:
Fixed